packages/amplify-provider-awscloudformation/src/print-cdk-migration-warning.ts

```diff
@@ -0,0 +1,112 @@
+import {
+  $TSContext, AmplifyCategories, AmplifyNodePkgDetectorProps, AmplifyNodePkgDetector, IAmplifyResource, pathManager,
+} from 'amplify-cli-core';
+import { printer } from 'amplify-prompts';
+import * as fs from 'fs-extra';
+import _ from 'lodash';
+import * as path from 'path';
+
+/**
+ * type to print cdk warning message
+ */
+export type AmplifyWarning = {
+  impactedFiles: string[],
+  resolutionMessage: string
+}
+/**
+ * print cdk migration warning if required
+ */
+export const printCdkMigrationWarning = async (context: $TSContext): Promise<void> => {
+  const resourcesToBuild: IAmplifyResource[] = [];
+  const { allResources } = await context.amplify.getResourceStatus();
+  allResources.forEach(resource => {
+    resourcesToBuild.push({
+      service: resource.service as string,
+      category: resource.category as string,
+      resourceName: resource.resourceName as string,
+    });
+  });
+  // check for override.ts file enabled
+  const migrationString = getMigrationMessage(resourcesToBuild);
+  if (!_.isEmpty(migrationString)) {
+    printer.warn(migrationString);
+  }
+};
+
+const getOverridesWarningObj = (resourcesToBuild: IAmplifyResource[]): AmplifyWarning | undefined => {
```

Comment:
nit: please drop `Obj` suffix from all these methods. We know it's object by looking at return type.